### PR TITLE
Capstone

### PR DESCRIPTION
**Pull request by franciscoPonceDev**
# Microverse Module 1 Capstone

> In this capstone project I've built a website for a conference. Microverse provided some [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) for me to create the website, but I've personalized the content. 

## Author Notes

> I've styled the website with a background that creates a sense od depth. I really liked it that way. If its mandatory that we use the same background color in the sections I'll do that, but if it's not, I REALLY liked the manga theme background. It goes well with the theme I've chose to this project.

### Project requirements

- Personalize the content of your page. Choose a topic that is different than the one in the original design.
- Follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
  - Colors.
  - Typography: font face, size and weight.
  - Layout: composition and space between elements.
- The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- You can use Bootstrap for styling, if you want to.
- You should build only these 2 pages:
  - The *home page*.
  - The *about page*.
- Each of these pages should have versions for 2 different screen sizes: 
  - Mobile: up to 768px wide.
  - Desktop: 768px or wider.

- Interactions
  - Links
    - The *home page* should have a link in the menu to the *about page*.
    - The logo in the header links to the *home page*.
  - Mobile menu
    - When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
    - There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
    - The mobile menu should have a close (X) button that closes the menu.
- Dynamic page
  - The section "Featured speakers" should be created dynamically in JavaScript.
  - You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+      </div>
+      <button
+        type="button"
+        class="btn btn-outline-secondary btn-floating btn-lg"
+        id="btn-back-to-top"
+        >
+        <i class="fas fa-arrow-up"></i>
+      </button>
+    </section>
+    <section id="program" class="program">
```
Comment (Hamzaoutdoors) [inline index.html:73]: - [x] In this `section` you have a small issue with the design as you can see in the screenshots below, the program title is overflowing over the program card, Kindly fix this issue 

**Current mobile menu**           |  **Original design**
:-------------------------:|:-------------------------:
  |

**Review by Hamzaoutdoors — CHANGES_REQUESTED** [review]
Hey @franciscoPonceDev, I am **Hamza** your reviewer for this PR 😸 

_You did a great job Francisco, I really enjoyed surfing on your fancy website 😃_ 

## To Highlight
✔️ The repo looks professional
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ Wonderful UI design and good choice of theme🥇 
✔️ Great Job building your project with **Mobile-First**

There are some issues that you still need to work on to go to the next project but you are almost there!

image

## Required Changes ♻️

_See the inline comments below for more required changes 😃_  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OTIOANAL ] 

- In your `CSS` file I can see that selector classes are used multiple times and rules are repeated as you can see in the screenshot below (I used this [Link](https://courses.cs.washington.edu/courses/cse154/20su/resources/assets/css-redundancy-checker/index.html) to check for redundancy. Because it's a good practice to keep your code `DRY` (Don't Repeat Yourself, Principe) I recommend you to use **Bootstrap** classes as you already add the **Bootstrap** framework to your project
- I recommend you too to regroup your `JavaScript` and `CSS` files in folders i.e. `script` and `styles` folder, with that your root project will be more organized 😸 
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by franciscoPonceDev** [comment]
Thank you very much for your suggestions. This Time, I'll only implement the obligatory required changes, but next project I'll pay attention to doing as you mentioned in the optional suggestions! :)

**Review by leolpaz — APPROVED** [review]
Hi @franciscoPonceDev , 

Your project is complete!
Congratulations! 🎉🎉 

To highlight:

- Congratulations on implementing the changes required by the previous reviewer

Now there is nothing left to do but to merge it! 🏆 





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**